### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by Priyanshi-Tiwari**
- Added initial project setup using CRA and Mockbee.
- Added Mockman.
- Added folder structure and placeholder components.
- Configured Routing.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
+.App {
```
Comment (coldpigli) [inline src/App.css]: you can remove styles that you are not using, created by "create-react-app"

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
```
Comment (coldpigli) [inline src/App.css]: same. you can remove it since you're not using it anywhere

**Review by coldpigli — APPROVED** [review]
Suggested minor changes

**Review by therajatg — APPROVED** [review]
Other than removing the un-necessary, code looks good

File: src/App.css (hunk 1 above)
Comment (Priyanshi-Tiwari) [reply src/App.css]: okay...will do it

**Pull request merged**